rcp/org.testeditor.rcp4.tcltestrun.tests/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncherTest.xtend

```diff
@@ -0,0 +1,48 @@
+package org.testeditor.rcp4.tcltestrun
+
+import java.io.File
+import java.nio.file.Files
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.runtime.IPath
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.runners.MockitoJUnitRunner
+
+import static org.junit.Assert.*
+import static org.mockito.Mockito.*
+
+@RunWith(MockitoJUnitRunner)
+class TclMavenLauncherTest {
+
+	@Mock
+	MavenExecutor executor
+
+	@InjectMocks
```

Comment:
my personal style only - I prefer the following:
```
   @InjectMocks TclMavenLauncher launcher
   @Mock MavenExecutor executor
```

(first the class under test, then the mocks)